synanno/routes/opendata.py

```diff
@@ -249,18 +259,31 @@ def upload_file() -> Template:
                     else -1
                 )
```

Comment:
As you have removed the original logic of the view style that uses the bounding box, you can also remove the calculation of the partial volume.